x-pack/plugin/sql/qa/src/main/resources/geo/geosql.csv-spec

```diff
@@ -102,3 +102,32 @@ SELECT COUNT(city) count, CAST(SUBSTRING(ST_ASWKT(location), 8, 1) = '-' AS STRI
 9              |false
 6              |true
 ;
+
+selectFakeCoordinatesFromCityAndRegion
+SELECT region, city, ST_ASWKT(ST_WKTTOSQL(CONCAT(CONCAT(CONCAT(CONCAT('POINT (', CAST(LENGTH(city) AS STRING)), ' '), CAST(LENGTH(region) AS STRING)), ')'))) loc  FROM geo ORDER BY region, city;
```

Comment:
I don't get why you need this complexity, but I'm also unfamiliar with area.
Isn't it better to add real coordinate date to the table (index) ?

Also again you wrap it with a call to `ST_ASWKT`, why is that? Shouldn't GeoShape implement a `toString()` method so that it returns a human readable string as requited?

Reply:
Yes. It looks pretty ridiculous, I agree. I am going to significantly simplify it in the next iteration when I add some more useful functions that actually do something with shapes. So, I for now I need to work around lack of functions and inability to use cast because of #34557 and #33361